android/src/main/java/com/clevertap/rnsignedcallandroid/internal/util/InitConfigSerializer.kt

```diff
@@ -82,6 +90,42 @@ object InitConfigSerializer {
     }
   }
 
+  @JvmStatic
+  @Throws(Exception::class)
+  fun getM2PConfigurationFromReadableMap(readableMap: ReadableMap): M2PConfiguration? {
+    val m2pNotificationClickListener = M2PNotificationClickListener { context, m2pCallOptions ->
```

Comment:
@Anush-Shand How do we expect host app to pass the m2pNotificationClickListener and m2pCancelCtaClickListener? 
Would have to check the example app side of implementaion

Reply:
Discussed and addressed
We will fire the event from native every time. Now if the client attaches a listener they will get the callback otherwise not